swri_roscpp/CMakeLists.txt

```diff
@@ -14,16 +14,24 @@ find_package(rosidl_default_generators REQUIRED)
 find_package(std_msgs REQUIRED)
 find_package(std_srvs REQUIRED)
 
-# TODO pjr What to do about swri_roscpp-extras.cmake?
 add_library(${PROJECT_NAME} INTERFACE)
+target_compile_features(${PROJECT_NAME} INTERFACE cxx_std_17)
```

Comment:
You set the CMAKE_CXX_STANDARD above for 14. Should that be left as is or changed?